package.json

```diff
@@ -32,16 +32,18 @@
   },
   "homepage": "http://gpii.net",
   "dependencies": {
+    "ajv": "^1.4.9",
+    "dedupe-infusion": "1.0.0",
     "first-discovery": "git+https://github.com/GPII/first-discovery.git",
```

Comment:
Use of raw git urls binding to a project master is hazardous and can allow a project to break without warning. Please bind against particular revisions of these projects instead